guides/contributors/README.md

```diff
@@ -0,0 +1,99 @@
+# Contributors Guide
+
+## Index
+
+<ol>
+  <li>
+    <a href="#understanding-the-specification">Understanding the specification</a>
+    <ol>
+      <li>
+        <a href="#basic-concepts">Basic concepts</a>
+      </li>
+      <li>
+        <a href="#areas-of-the-spec">Areas of the spec</a>
+      </li>
+      <li>
+        <a href="#what-does-ref-do">What does $ref do?</a>
+      </li>
+      <li>
+        <a href="#the-parts">JSON Schema</a>
+      </li>
+    </ol>
+  </li>
+  <li>
+    <a href="#tooling-landscape-and-plans">Tooling landscape and plans</a>
+    <ol>
+      <li>
+        <a href="#the-parser">The parser</a>
+      </li>
+      <li>
+        <a href="#the-generator">The generator</a>
+      </li>
+      <li>
+        <a href="#the-editor">The editor</a>
+      </li>
+      <li>
+        <a href="#tooling-plans">Tooling plans</a>
+      </li>
+    </ol>
+  </li>
+  <li>
+    <a href="#contributing-processes">Contributing processes</a>
+    <ol>
+      <li>
+        <a href="#task-management">Task management</a>
+        <ol>
+          <li>
+            <a href="#kanban-board">Kanban board</a>
+          </li>
+          <li>
+            <!-- Labels, epics, milestones, and releases -->
+            <a href="#creating-a-task">Creating a task</a>
+          </li>
+          <li>
+            <a href="#picking-a-task">Picking a task</a>
+          </li>
+          <li>
+            <a href="#working-on-a-task">Working on a task</a>
+          </li>
+          <li>
+            <a href="#definition-of-done">Definition of done</a>
+          </li>
+        </ol>
+      </li>
+      <li>
+        <a href="#coding">Coding</a>
+        <ol>
+          <li>
+            <a href="#pushing-early">Pushing early</a>
+          </li>
+          <li>
+            <a href="#reviewing-pull-requests">Reviewing pull requests</a>
+          </li>
+          <li>
+            <a href="#merging-your-code">Merging your code</a>
+          </li>
+        </ol>
+      </li>
+      <li>
+        <a href="#communication">Communication</a>
+        <ol>
+          <li>
+            <!-- Keep communication public when possible, be nice, other may not have enough context, do not accuse people, look for solutions instead of culprits, we're a team, don't try to impose your opinion, give voice to shy people, etc. -->
```

Comment:
"...be professional, courteous, kind since others may not have enough..."

Reply:
Awesome! Thanks!